examples/azure_auth/entity/post.rs

```diff
@@ -46,7 +46,7 @@ impl Related<super::comment::Entity> for Entity {
 
 impl ActiveModelBehavior for ActiveModel {}
 
-#[derive(Debug, Clone, PartialEq, EnumIter, DeriveActiveEnum, Deserialize, Serialize, DeriveActixAdminEnumSelectList)]
+#[derive(Debug, Clone, PartialEq, EnumIter, DeriveDisplay, DeriveActiveEnum, Deserialize, Serialize, DeriveActixAdminEnumSelectList)]
```

Comment:
Once the main error was fixed I got errors about these.

```
error[E0599]: `Tea` doesn't implement `std::fmt::Display`
   --> tests/test_setup/post.rs:8:123
    |
8   | #[derive(Clone, Debug, PartialEq, DeriveEntityModel, Deserialize, Serialize, DeriveActixAdmin, DeriveActixAdminViewModel, DeriveActixAdminModel, DeriveActixAdminModelSelectList)]
    |                                                                                                                           ^^^^^^^^^^^^^^^^^^^^^ `Tea` cannot be formatted with the default formatter
...
51  | pub enum Tea {
    | ------------
    | |
    | method `to_string` not found for this enum
    | doesn't satisfy `Tea: ToString`
    | doesn't satisfy `Tea: std::fmt::Display`
    |
    = note: the following trait bounds were not satisfied:
            `Tea: std::fmt::Display`
            which is required by `Tea: ToString`
note: the trait `std::fmt::Display` must be implemented
   --> /Users/keith/.asdf/installs/rust/1.71.1/toolchains/1.71.1-x86_64-apple-darwin/lib/rustlib/src/rust/library/core/src/fmt/mod.rs:648:1
    |
648 | pub trait Display {
    | ^^^^^^^^^^^^^^^^^
    = note: this error originates in the derive macro `DeriveActixAdminModel` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0599]: `Tea` doesn't implement `std::fmt::Display`
   --> tests/test_setup/post.rs:49:87
    |
49  | #[derive(Debug, Clone, PartialEq, EnumIter, DeriveActiveEnum, Deserialize, Serialize, DeriveActixAdminEnumSelectList)]
    |                                                                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `Tea` cannot be formatted with the default formatter
50  | #[sea_orm(rs_type = "String", db_type = "Enum", enum_name = "tea")]
51  | pub enum Tea {
    | ------------
    | |
    | method `to_string` not found for this enum
    | doesn't satisfy `Tea: ToString`
    | doesn't satisfy `Tea: std::fmt::Display`
    |
    = note: the following trait bounds were not satisfied:
            `Tea: std::fmt::Display`
            which is required by `Tea: ToString`
note: the trait `std::fmt::Display` must be implemented
   --> /Users/keith/.asdf/installs/rust/1.71.1/toolchains/1.71.1-x86_64-apple-darwin/lib/rustlib/src/rust/library/core/src/fmt/mod.rs:648:1
    |
648 | pub trait Display {
    | ^^^^^^^^^^^^^^^^^
    = note: this error originates in the derive macro `DeriveActixAdminEnumSelectList` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Which lead me to https://github.com/SeaQL/sea-orm/pull/1726 where I saw that they pulled out the default `Display` impl.